### DISCUSSION TITLE
POS Printing STAR TSP100 Bluetooth printer from Android app

### DISCUSSION THREAD

**Discussion by datadumped** [lukevp/ESC-POS-.NET#209] [Q&A]
Getting started with learning .Net Maui and my target platform is Android. I'm able to print to the STAR TSP100 POS printer when I run it from a Windows device. The printer is connected via Bluetooth set on COM4. No issues here.

var printer = new SerialPrinter(portName: "COM4", baudRate: 115200);

When I run the Android emulator (on my Samsung tablet), it doesn't recognize COM4.

I'm not sure how to connect to the printer from Android. I tried putting in the device name, but that didn't work.
Please help.

**Comment by Cruik** [comment]
You still need help here ?
